chirpus-2/index.css

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400&family=Roboto&display=swap');
+body {
+    margin: 10px;
+    background-color: #f9fafc;
+    font-family: 'Open-Sans', sans-serif;
+}
+.nav {
+    display: flex;
+    position: relative;
```

Comment:
The position relative on .nav and position: absolute on the .nav__logo started throwing you for a loop I think. You could remove lines 9, 12, 15, 16, 17, and 22.